### PR TITLE
Replace Bento by Bootstrap for attributes

### PR DESCRIPTION
**Pull request by dmarcoux**

### CODE REVIEW

**Comment by dmarcoux** [comment]
[Webui::AttributeController](https://trello.com/c/pBhh8T91/1245-webuiattributecontroller)

File: src/api/app/controllers/webui/attribute_controller.rb [hunk 1]
```diff
@@ -25,6 +23,8 @@ def new
 
     authorize @attribute, :create?
 
+    @attribute_types = AttribType.includes(:attrib_namespace).all.sort_by(&:fullname)
+
     switch_to_webui2
```
Comment (hennevogel) [inline src/api/app/controllers/webui/attribute_controller.rb]: Hm, why keep this around?

Comment (dmarcoux) [reply src/api/app/controllers/webui/attribute_controller.rb]: It's loading the old assets if I remove it.

Comment (DavidKang) [reply src/api/app/controllers/webui/attribute_controller.rb]: We still need this, because the new layout is not integrated.

**Pull request merged**